eclipse-theia/theia-blueprint
# Theia 1.27.0, Jenkins Update

#### What it does
* Updates Theia to 1.27 and upgrades other dependencies. 
* Also integrates the Jenkinsfile changes done and tested on the next branch (https://github.com/eclipse-theia/theia-blueprint/pull/220) into the main Jenkinsfile. 
* Adds retry to the "Sign and Upload Windows"-step as well, since parts are executed on Linux: https://ci.eclipse.org/theia/job/Theia%20PRs/job/PR-223/


#### How to test
Build Theia-Blueprint and check that application starts up and works as expected. 

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
